packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -965,15 +923,15 @@ class _TableState<T> extends State<_Table<T?>> with AutoDisposeMixin {
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.stretch,
           children: [
-            TableRow.tableHeader(
+            TableRow<T>.tableHeader(
               key: const Key('Table header'),
               linkedScrollControllerGroup:
                   _linkedHorizontalScrollControllerGroup,
-              columns: widget.columns,
+              columns: widget.columns as List<ColumnData<T>>,
               columnWidths: widget.columnWidths,
-              sortColumn: sortColumn,
+              sortColumn: sortColumn as ColumnData<T>?,
```

Comment:
we should be able to get rid of the cast if we type sortColumn correctly where it is defined. T should be non-nullable where sortColumn is defined.

Reply:
done